{{cookiecutter.project_shortname}}/{{cookiecutter.package_name}}/config.py

```diff
@@ -87,6 +87,15 @@ def _(x):
 #: client. Set to False, in case of doubt.
 ACCOUNTS_USERINFO_HEADERS = True
 
+# Sessions
+# ========
+#: SameSite cookie type to be used to store user sessions.
+SESSION_COOKIE_SAMESITE = 'Strict'
+
+#: Session maniputalion interface (Flask-KVSession).
+SESSION_INTERFACE_FACTORY = \
+    'invenio_accounts.sessions:KVSessionInterfaceStoreAuthenticated'
```

Comment:
[Initially decided to be set by Invenio-Accounts](https://github.com/inveniosoftware/invenio-accounts/pull/292/commits/be9bd3d412106bdaed5ff9112ee6f2f0ed8057c3#diff-dc931617b1818fe4b91115eb4fd23683R272) but there is no guarantee that Invenio-Accounts will be loaded before Flask-KVSession-Invenio.

Reply:
How is Flask-KVSession-Initialised? I think there's potentially something if it's loaded before Invenio-Accounts

Reply:
I had an extra check - it's because the variables are filtered out [here](https://github.com/inveniosoftware/invenio-accounts/blob/master/invenio_accounts/ext.py#L231-L237).

Reply:
Ups, wrong highlight - [here](https://github.com/inveniosoftware/invenio-accounts/blob/master/invenio_accounts/ext.py#L239-L242)